kafka/protocol/metadata.py

```diff
@@ -52,17 +52,15 @@ class MetadataRequest_v0(Struct):
     API_VERSION = 0
     RESPONSE_TYPE = MetadataResponse_v0
     SCHEMA = Schema(
-        ('topics', Array(String('utf-8'))) # Empty Array (len 0) for all topics
+        ('topics', Array(String('utf-8')))  # Empty Array (len 0) for all topics
```

Comment:
Should this have additional class vars?

    ALL_TOPICS = None  # Empty Array (len 0) for all topics